Birthday-2024-Project/Scripts/ScreenManagement/ScreenLogic.gd

```diff
@@ -0,0 +1,37 @@
+class_name ScreenLogic
```

Comment:
ScreenLogic is not initalized in main menu when we go back to it from Play or Win state which causes game to crash once you click Settings or Exit

Reply:
Navigating back to Main Menu now uses the updated method. Issue is now fixed